_includes/structure/footer.html

```diff
@@ -1,42 +1,50 @@
-<!--- Footer stuff -->
-<div id="footer-layout" role="contentinfo" class="l-footer">
-  <div class="footer-holder">
-    <div class="region--copyright region region-copyright">
-      <div id="block-sitebranding" class="col-md-4">
-        <div id="site-branding">
-          <div id="logo">
-            <a href="https://museums.cam.ac.uk/" title="Home" rel="home" class="site-logo">
-              <img src="{{ site.url }}{{ site.baseurl }}/images/logos/logo.svg" alt="Home">
-            </a>
-          </div>
-        </div>
-      </div>
-      <div id="block-acelogo" class="col-md-8 block block-block-content ">
-        <div class="content">
-          <div class="clearfix text-formatted field field--name-body field--type-text-with-summary field--label-hidden field__item">
-            <p>
-              <a href="https://www.artscouncil.org.uk/"><img alt="ACE website logo"  src="{{ site.url }}{{ site.baseurl }}/images/logos/u43.png"></a>
-            </p>
-            <p>&nbsp;</p>
-          </div>
-        </div>
-      </div>
+<div class="bg-white border-cambridge">
+  <div class="fitz-footer">
+    <div class="text-center">
+      <a href="https://fitzmuseum.cam.ac.uk/">
+        <img class="mx-auto my-2" loading="lazy" width="200" height="62" alt="University of Cambridge Museums logo"
```

Comment:
If this is a picture of the Fitzwilliam logo, then the alt text here should be changed to reflect that ideally.